test/differential_equations.jl

```diff
@@ -0,0 +1,16 @@
+using StochasticRounding
+using Test
+using DifferentialEquations
+
+
+
+@testset "Successful run with DE.jl" begin
+    try
+        f = (u,p,t) -> (p*u)
+        prob_ode_linear = ODEProblem(f,Float32sr(1.0)/Float32sr(2.0),(Float32sr(0.0),Float32sr(1.0)),Float32sr(1.01));
+        sol =solve(prob_ode_linear,Tsit5())
+        @test true # The default completes without any errors 
```

Comment:
You can skip this line as it suggests there was a test but there actually wasn't, an error in any of the above will also make the CI fail.